cmake/modules/AddSwift.cmake

```diff
@@ -75,6 +75,36 @@ function(_set_target_prefix_and_suffix target kind sdk)
   endif()
 endfunction()
 
+function(_add_host_variant_swift_sanitizer_flags target)
+  if (LLVM_USE_SANITIZER)
```

Comment:
I really want to say, please use LLVM style ...

```cmake
  if(NOT LLVM_USE_SANITIZER)
    return()
  endif()
```

That way we can unindent the rest of the implementation.  I'll let you decide on that.